plugins/data-management/src/PluginStoreWidget/components/PluginStoreWidget.tsx

```diff
@@ -154,16 +154,13 @@ function PluginStoreWidget({ model }: { model: PluginStoreModel }) {
         ) : pluginArray ? (
           pluginArray
             .filter(plugin => {
-              // If pugin only has cjsUrl, don't display outside desktop
-              if (
-                !isElectron &&
-                !(plugin.esmUrl || plugin.url || plugin.umdUrl)
```

Comment:
afaik this would filter out e.g. esmLoc,umdLoc which we do not want. note we do not have a cjsLoc...not sure if we need it